src/controllers/itemDetails/index.js

```diff
@@ -666,10 +666,12 @@ function reloadFromItem(instance, page, params, item, user) {
     if (item.Type == 'Person' && item.ProductionLocations && item.ProductionLocations.length) {
         let location = item.ProductionLocations[0];
         if (!layoutManager.tv && appHost.supports('externallinks')) {
-            location = `<a is="emby-linkbutton" class="button-link textlink" target="_blank" href="https://www.openstreetmap.org/search?query=${encodeURIComponent(location)}">${location}</a>`;
+            itemBirthLocation.innerHTML = globalize.translate('BirthPlaceValue', `<a is="emby-linkbutton" class="button-link textlink" target="_blank" href="https://www.openstreetmap.org/search?query=${encodeURIComponent(location)}"></a>`);
+            page.querySelector('#itemBirthLocation > a').innerText = `${location}`;
+        } else {
+            itemBirthLocation.innerText = globalize.translate('BirthPlaceValue', `${location}`);
         }
         itemBirthLocation.classList.remove('hide');
-        itemBirthLocation.innerText = globalize.translate('BirthPlaceValue', location);
```

Comment:
You can revert my changes in this line (https://github.com/jellyfin/jellyfin-web/commit/59adbc348a37bccc8f9a277378e863efbe95497a) and use `escapeHtml`:
```js
        let location = item.ProductionLocations[0];
        if (!layoutManager.tv && appHost.supports('externallinks')) {
            location = `<a is="emby-linkbutton" class="button-link textlink" target="_blank" href="https://www.openstreetmap.org/search?query=${encodeURIComponent(location)}">${escapeHtml(location)}</a>`;
        } else {
            location = escapeHtml(location);
        }
        itemBirthLocation.classList.remove('hide');
        itemBirthLocation.innerHTML = globalize.translate('BirthPlaceValue', location);
```

Reply:
fixed in 2521b0614225db9a6f6a063c2eccf754d74fb15f

Reply:
shouldn't the links in `renderLinks` (lines 745 and 750) also use `escapeHtml()`?

Reply:
I replied in the main thread to mark this conversation as resolved.